### PR TITLE
docs: deploy bytebase via rainbond

### PR DESCRIPTION
**Pull request by zzzhangqi**
Signed-off-by: Qi Zhang <smallqi1@163.com>

### CODE REVIEW

File: content/docs/get-started/install/deploy-to-rainbond.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: Deploy to Rainbond
+---
+
+This tutorial is for users who don't know Container and Kubernetes. 
+
+If you don't have Container and Kubernetes knowledge, And want to install Bytebase in Kubernetes, you can use Rainbond to deploy bytebase. [Rainbond](http://www.rainbond.com) is cloud native application management platform built on Kubernetes, easy to use, no need to know Kubernetes knowledge, easily deploy applications in Kubernetes.
+
+## Prerequisites
+
+You need to have a Rainbond. Please ref [Rainbond Quick Install](https://rainbond.com/docs/quick-start/quick-install).
```
Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:11]:
```suggestion
To install Rainbond, please refer to [Rainbond Quick Install](https://rainbond.com/docs/quick-start/quick-install).
```

File: content/docs/get-started/install/deploy-to-rainbond.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Deploy Bytebase
+
+After login in to Rainbond, click `Market` in the left menu, switch to open source app store, and search `Bytebase` in the search box, and click Install button.
```
Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:15]:
```suggestion
After logging in Rainbond, click `Market` in the left menu, switch to open source app store, and search `Bytebase` in the search box, and click the Install button.
```

File: content/docs/get-started/install/deploy-to-rainbond.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+![](/static/docs/get-started/install/rainbond-install.png)
```
Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:17]: Images need to be converted to webp format

File: content/docs/get-started/install/deploy-to-rainbond.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+Fill in the installation information, and click Confirm button startup install.
```
Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:19]:
```suggestion
Fill in the following information, and click `Confirm` button to install.
```

File: content/docs/get-started/install/deploy-to-rainbond.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+* Team: select a team or create a new team
+* Cluster: select a cluster
+* Application: select an application or create a new application
+* Version: select a version
+
+Moment later, Bytebase will be installed successfully, via the Access button to access Bytebase.
```
Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:26]:
```suggestion
After installation, Bytebase can be accessed via the `Access` button.
```

File: content/docs/get-started/install/deploy-to-rainbond.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+> Via Rainbond installed Bytebase, external PostgreSQL is used by default.
```
Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:28]: Let's used the hint block https://bytebase.com/docs/document-write-guide#hint

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by zzzhangqi** [comment]
@tianzhou All suggestion have been submitted. but cla failed to pass, I have signed cla.

**Comment by tianzhou** [comment]
> @tianzhou All suggestion have been submitted. but cla failed to pass, I have signed cla.

@zzzhangqi you need to add your name to https://github.com/bytebase/clabot-config/blob/main/.clabot

File: content/docs/get-started/install/deploy-to-rainbond.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
+title: Deploy to Rainbond
+---
+
+This tutorial is for users who don't know Container and Kubernetes. 
+
+If you don't have Container and Kubernetes knowledge, And want to install Bytebase in Kubernetes, you can use Rainbond to deploy bytebase. [Rainbond](http://www.rainbond.com) is cloud native application management platform built on Kubernetes, easy to use, no need to know Kubernetes knowledge, easily deploy applications in Kubernetes.
```
Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:7]:
```suggestion

If you are unfamiliar with Kubernetes, and want to install Bytebase in Kubernetes, you can use Rainbond to deploy. [Rainbond](http://www.rainbond.com) is a cloud-native application management platform built on Kubernetes and simplifies the application deployment to Kubernetes.
```

File: content/docs/get-started/install/deploy-to-rainbond.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+* Application: select an application or create a new application
+* Version: select a version
+
+After installation, Bytebase can be accessed via the `Access` button.
+
+<hint-block type="info">
+
+Via Rainbond installed Bytebase, external PostgreSQL is used by default. 
+
+External URL for Bytebase is provided by Rainbond by default, You can go to the Bytebase component -> Environment configuration, Modify `EXTERNAL_URL` Environment Variables to customize the external URL.
```
Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:32]:
```suggestion
Rainbond uses [--external-url](/docs/get-started/install/external-url) to store the Bytebase metadata in the external PostgreSQL. You can go to the Bytebase component -> Environment configuration, Modify `EXTERNAL_URL` Environment Variables to customize the external URL.
```

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/get-started/install/deploy-to-rainbond.md (hunk 8 above)
Comment (zzzhangqi) [reply content/docs/get-started/install/deploy-to-rainbond.md:32]: This is two paragraphs.

Rainbond uses [--pg](/docs/get-started/install/external-postgres) to store the Bytebase metadata in the external PostgreSQL.

Rainbond uses [--external-url](/docs/get-started/install/external-url) to provides external access to Bytebase. You can go to the Bytebase component -> Environment configuration, Modify `EXTERNAL_URL` Environment Variables to customize the external URL.

Comment (tianzhou) [reply content/docs/get-started/install/deploy-to-rainbond.md:32]: My bad. Got drunk

Comment (tianzhou) [inline content/docs/get-started/install/deploy-to-rainbond.md:32]:
```suggestion
Rainbond uses [--pg](/docs/get-started/install/external-postgres) to store the Bytebase metadata in the external PostgreSQL.

Rainbond uses [--external-url](/docs/get-started/install/external-url) to provide external access to Bytebase. You can go to the Bytebase component -> Environment configuration, and Modify EXTERNAL_URL Environment Variables to customize the external URL.
```

Comment (tianzhou) [reply content/docs/get-started/install/deploy-to-rainbond.md:32]: @zzzhangqi please see the new edit, but I have a question, do we provide the similar ENV variable for configuring external PG?

Comment (zzzhangqi) [reply content/docs/get-started/install/deploy-to-rainbond.md:32]: Yes, provide. `POSTGRES_HOST` `POSTGRES_PORT` `POSTGRES_DB` `POSTGRES_USER` `POSTGRES_PASSWORD`

**Review by tianzhou — COMMENTED** [review]
Please take a look at the suggested edit

**Comment by zzzhangqi** [comment]
@tianzhou All suggestion have been submitted.

File: content/docs/_layout.md [hunk 9]
```diff
@@ -25,6 +25,8 @@ expand_section_list: ["Introduction", "Get Started"]
 
 #### [Option 5 Build from Source Code](/get-started/install/build-from-source-code)
 
+#### [Option 6 Deploy to Rainbond](/get-started/install/deploy-to-rainbond)
```
Comment (tianzhou) [inline content/docs/_layout.md:28]: let's make this option 4

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by zzzhangqi** [comment]
@tianzhou All suggestion have been submitted.

**Comment by tianzhou** [comment]
> @tianzhou All suggestion have been submitted.

Thanks for the contribution. Looking forward to the collaboration between Rainbond and Bytebase!

**Pull request merged**